ci-operator/step-registry/upi/install/vsphere/upi-install-vsphere-commands.sh

```diff
@@ -73,9 +73,12 @@ function gather_bootstrap_and_fail() {
     GATHER_BOOTSTRAP_ARGS+=('--master' "${control_plane_0_ip}" '--master' "${control_plane_1_ip}" '--master' "${control_plane_2_ip}")
 
     set -e
-    openshift-install --dir=/tmp/artifacts/installer gather bootstrap --key "${SSH_PRIV_KEY_PATH}" "${GATHER_BOOTSTRAP_ARGS[@]}"
-
-  return 1
```

Comment:
Are we intentionally removing this `return 1`? From the name of the `gather_bootstrap_and_fail` function, the return seems intentional, although I cannot see how it is useful. Should we change the name of the function if we are going to stop returning 1?

Reply:
@staebler with the `return 1` (unless it was something else - maybe the openshift-install gather bootstrap failing?) the entire script exited before the copy would run.
maybe the `set -e` just needs to be moved after?

Reply:
After chatting with @rvanderp3 retested including `return 1` but removing `set -e`. Works as expected.

I think the return was a requirement from the previous template when we did:
https://github.com/openshift/release/blob/master/ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml#L2117

template `gather_bootstrap_and_fail`:
https://github.com/openshift/release/blob/master/ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml#L380

Reply:
Well if we don't need the `return 1`, let's not leave it in there, particularly now that the name of the function does not stipulate that it will fail.